src/Components/NewBlog.js

```diff
@@ -1,41 +1,61 @@
-import axios from 'axios';
-import React, { useState } from 'react';
+import axios from "axios";
+import React, { useState } from "react";
 
-const CreatePost = () =>{
-  const [name, setName] = useState("");
-  const [title, setTitle] = useState("");
-  const [content, setContent] = useState("");
-  console.log(name);
+const CreatePost = () => {
+  const [post, setPost] = useState({
+    name: "",
+    title: "",
+    content: "",
+  });
 
-  const handleSubmit = (e) => {
+  const handleSubmit = async (e) => {
     e.preventDefault();
-    axios.post("https://blogpostapi1.herokuapp.com/", {
-      name,
-      title,
-      content,
-      }
-    )
+    try{await axios.post("https://blogpostapi1.herokuapp.com/", {
+      name:post.name,
+      title:post.title,
+      content:post.content
+    });
+  }catch(e){
+    console.log("Post Error")
+    }
   };
-  
-  return(
+
+  return (
     <form onSubmit={handleSubmit}>
       <div className="field">
-        <label for="author">Author:
-          <input type="text" id="author" onChange={(e)=> setName(e.target.value)}/>
+        <label for="author">
+          Author:
+          <input
+            type="text"
+            id="author"
+            value={post.name}
+            onChange={(e) => setPost({ ...post, name: e.target.value })}
+          />
         </label>
       </div>
       <div className="field">
-        <label for="title">Title:
-          <input type="text" id="title" onChange={(e)=> setTitle(e.target.value)}/>
+        <label for="title">
+          Title:
+          <input
+            type="text"
+            id="title"
+            value={post.title}
+            onChange={(e) => setPost({ ...post, title: e.target.value })}
```

Comment:
D'you remember that conversation we had about using the setState setter function style?

Please use that here